pkg/asset/store/store_test.go

```diff
@@ -390,7 +390,7 @@ func TestStoreFetchIdempotency(t *testing.T) {
 	tempDir := t.TempDir()
 
 	for i := 0; i < 2; i++ {
-		store, err := newStore(tempDir)
+		store, err := newStore(tempDir, false)
```

Comment:
Lint is complaining here as new changes call `Validate` separately (i.e. no such flag anymore) :sweat_smile:

Reply:
The latest run: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_installer/9225/pull-ci-openshift-installer-main-golint/1914339061917552640